src/Init/MetaTypes.lean

```diff
@@ -68,45 +68,115 @@ namespace Simp
 
 def defaultMaxSteps := 100000
 
+/--
+The configuration for `simp`.
+Passed to `simp` using, for example, the `simp (config := {contextual := true})` syntax.
+
+See also `Lean.Meta.Simp.neutralConfig`.
+-/
 structure Config where
+  /--
+  The maximum number of subexpressions to visit when performing simplification.
+  The default is large number, but not infinite.
```

Comment:
```suggestion
  The default is 100000.
```
Giving the actual number helps with giving people a sense of e.g. it's order of magnitude.